src/classes/NPSP_Address.cls

```diff
@@ -476,16 +449,75 @@ public inherited sharing class NPSP_Address {
                 addr.Seasonal_End_Month__c != null && addr.Seasonal_End_Day__c != null);
     }
 
-    public void mailingState(String state) {
+    public void state(String state) {
         addr.MailingState__c = state;
     }
 
-    public String mailingState() {
+    public String street() {
+        return addr.MailingStreet__c;
+    }
+
+    public String street2() {
+        return addr.MailingStreet2__c;
+    }
+
+    public String city() {
+        return addr.MailingCity__c;
+    }
+
+    public String state() {
         return addr.MailingState__c;
     }
 
-    public String mailingCountry() {
+    public String postalCode() {
+        return addr.MailingPostalCode__c;
+    }
+
+    public String country() {
         return addr.MailingCountry__c;
     }
 
+    public Decimal longitude() {
+        return addr.Geolocation__Longitude__s;
+    }
+
+    public Decimal latitude() {
+        return addr.Geolocation__Latitude__s;
+    }
+
+    /*******************************************************************************************************
+    * @description utility to combine mailing street fields into a single string with embedded CR LF
+    * @param addr the Address to look at
+    * @return boolean
+    */
+    public String multilineStreet() {
+        Boolean isOneLineStreetAddress = String.isBlank(addr.MailingStreet2__c);
+        if (isOneLineStreetAddress) {
+            return addr.MailingStreet__c;
+        }
+        return addr.MailingStreet__c + '\r\n' + addr.MailingStreet2__c;
+    }
+
+    /*******************************************************************************************************
+    * @description utility to compare two address objects address fields
+    * @param addrNew the new version of the Address
+    * @param addrOld the old version of the Address
+    * @param includeAddressType whether to include comparing the AddressType field
+    * @return boolean. true if any of the fields fields have changed on the Address
+    */
+    public Boolean isAddressChanged(Address__c addrOld, Boolean includeAddressType) {
```

Comment:
Boolean in a function argument here that we may be able to refactor. There doesn't appear to be anywhere that passes `true` to that argument - would that be code we could remove possibly?

Reply:
Yep, great call, Ben!  It looks like we might be able to adjust the one call site for this method and have it use the isEqualTo method instead, which is a dupe of this method.  I'll take a look!

Reply:
Was able to remove this method entirely.  Thanks, Ben!

Reply:
@kenlewis-sfdo - would be curious to hear if the refactored work would eliminate this Bug we recently created that does use isAddressChanged() with a value of true during one scenario:
#6416